### ISSUE TITLE
Bundle install gives "Source does not contain any versions of 'spree-html-email'"

### ISSUE THREAD

**Issue by binaryphile** [joshnuss/spree-html-email#4]
Add to Gemfile using :git => 'git://github.com/joshnuss/spree-html-email.git'.

Bundle install and you get the described error.

**Comment by msevestre** [comment]
Yes I have the same issue. Any idea what that means?

**Comment by joshnuss** [comment]
What version of spree?

**Comment by binaryphile** [comment]
Spree 1.1, 1-1-stable branch.

**Comment by msevestre** [comment]
Spree 1-2 as well ....it's more of a gem config issue i believe

**Comment by msevestre** [comment]
well issue is simple 
In the gemspec file, it should read 
  s.name        = 'spree-html-email' instead of   s.name        = 'spree_html_email'

**Comment by msevestre** [comment]
it's corrected here c9452fe

**Comment by msevestre** [comment]
Actually the problem is that the name of the gem is not consistent. It should be spree_html_email
One needs to do the following
gem "spree_html_email", :path => '../spree_extensions/spree-html-email' and it works. Forget the PR

**Comment by binaryphile** [comment]
The problem is that there is no readme with such instructions.

**Comment by msevestre** [comment]
yep read me should be updated or better, gem name changed so that it
matches spree/ruby convention

**Comment by binaryphile** [comment]
Sadly, it is the convention, it's just a bad one.  From the spree docs for creating extensions:

> It is convention to use the spree- naming convention for your GitHub repository and spree_ for your gem name. So for example, if you are creating a “foofah” extension the GitHub project would be named spree-foofah and the gem would be spree_foofah.

It's just a terrible convention because it's nonintuitive (for the two of us, at least) and causes confusion when there isn't documentation to remind us. :)

I think most extensions wisely ignore the convention, so I agree with your solution though.

**Comment by msevestre** [comment]
That's interesting. Almost all extensions out there are not following this
rule. (even the one officially supported by spree itself)
Well good to know ))
